### DISCUSSION TITLE
The numbers in prompt doesn't seem to work. And what is the preffered tool for LoRA training?

### DISCUSSION THREAD

**Discussion by oldhand7** [lllyasviel/Fooocus#3451] [Q&A]
Hi, I set the prompt as "classroom with 5 desk, 10 students" 
I expect the result containing exact numbers of desks, students but not.
Could you tell me how to resolve this?

What is the preffered tool to train LoRA? Kohya or stable diffusion webui?
Can we use trained model in Fooocus?
Thank you.

**Comment by mashb1t** [comment]
> I expect the result containing exact numbers of desks, students but not.
Could you tell me how to resolve this?

Increase prompt weight, but SD modeks we're never that good with exact counts of elements.

> What is the preffered tool to train LoRA? Kohya or stable diffusion webui?

You can't train a LoRA with SD WebUI afaik, only models.

> Can we use trained model in Fooocus?

Obv. yes, you're already doing so when using the default of Foooocus or any other LoRA (Juggernaut is a fine-tuned model based on SDXL)

**Reply by oldhand7** [reply]
Thank you.
Can I use Kohya for LoRA training?

**Reply by mashb1t** [reply]
That's what it's built foe and its sole purpose.
